LICENSE.txt

```diff
@@ -0,0 +1,675 @@
+AL PUBLIC LICENSE
```

Comment:
Seems to be missing some of [this](https://github.com/Islandora/islandora/blob/7f3d7819df4a9d010754e931ac4a318e0b3d81fb/LICENSE.txt#L1)...

... we sure this entirely the correct text?